### DISCUSSION TITLE
Knowing what state is currently set.

### DISCUSSION THREAD

**Discussion by graememk** [cmcgerty/Sofar2mqtt#5] [Ideas]
I wonder would it be possible to know what state we have currently set the inverter to?

For instance, In auto or battery save It could be in Standby, Charging or Discharging.

I know we could keep track of it locally, but It would be nice to get a feedback from the device that it actually got the message and was able to execute it successfully.

**Comment by zambujal** [comment]
there is the running_state, it does inform when in standby, charging or discharging, maybe I don't understand what you want.

**Reply by graememk** [reply]
If the Inverter is in Auto mode, the running state could be in standby, charging or discharging.

Other than the fact Ive set it to Auto, would I know its been put into the auto mode?

Ive implemented what i want in my own fork, but thought it could be useful for others and their automation.

**Comment by cmcgerty** [comment]
Thanks, I’ll take a look.


On 3 Oct 2021, at 13:18, graememk ***@***.***> wrote:

﻿

If the Inverter is in Auto mode, the running state could be in standby, charging or discharging.

Other than the fact Ive set it to Auto, would I know its been put into the auto mode?

Ive implemented what i want in my own fork, but thought it could be useful for others and their automation.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/cmcgerty/Sofar2mqtt/discussions/5#discussioncomment-1420753>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADFHULNAUDTS7JRYF4ESPR3UFBCXNANCNFSM4ZPY2RCQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by Banana5kin** [reply]
Anything come of this?
